sourcecode/driver.py

```diff
@@ -0,0 +1,36 @@
+from sourcecode.weatherByYearMonthBar.weatherman import WeatherManBar
+from sourcecode.weatherByYearMonthSingleBar.weatherman import WeatherManSingleBar 
+from sourcecode.weatherByYear.weatherman import WeatherManYear
+from sourcecode.weatherByYearMonth.weatherman import WeatherManYearMonth
+
+class Driver:
+
+    def __init__(self, directory_path):
+        """
+        Constructor for Driver
+
+        @params
+        :directory_path str: Path to folder of weather files
+        """
+        self.directory_path = directory_path
+
+    def getReport(self, option, date):
```

Comment:
PEP8 violation, function name should be in lower case with `_` separated i.e. `get_report `

Reply:
Return type annotation is missing for this function.